Scripts/script-CommonServer.yml

```diff
@@ -116,7 +116,7 @@ script: |-
           t.forEach(function(entry){
               var vals = [];
               headers.forEach(function(h){
-                  vals.push(!!entry[h] ? stringEscapeMD(formatCell(entry[h])) : ' ');
+                  vals.push((!!entry[h] || entry[h]==0) ? stringEscapeMD(formatCell(entry[h])) : ' ');
```

Comment:
I checked it again: '0G' == 0 returns false, while '00' == 0 returns true (obviously also '0'==0), none of them throw.